packages/access-om2/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# Copyright 2023 ACCESS-NRI
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+class AccessOm2(BundlePackage):
+    """ACCESS-OM2 bundle contains the coupled CICE5 and MOM5 models."""
+
+    homepage = "https://www.access-nri.org.au"
+
+    maintainers = ["harshula"]
+
+    version("master")
```

Comment:
GitHub has transitioned to using `main` as the name of the default branch, and this repo is using that convention. To be consistent we should use `main` as the version if the intention that it should use the current `main` branch.

Reply:
Good catch! In a bundle (SBD) `version` has a different meaning to an SPD. In an SPD, it's the version of the source package. In this case, I assume it's the version of the SBD. So should we use a date based version? e.g. "202305-1"

Reply:
Agreed calendar based versioning best, the [dev-docs suggest](https://github.com/ACCESS-NRI/dev-docs/wiki/Versioning) `2023.05.001`